crates/whiskers/src/context.rs

```diff
@@ -34,4 +37,30 @@ impl Context {
         }
         self.rng.gen_range(range)
     }
+
+    /// Helper function to generate a random boolean value
+    pub fn rng_boolean(&mut self) -> bool {
+        let num: f64 = self.rng.gen();
+        num > 0.5
+    }
+
+    /// Helper function to return a random item from a vector
+    pub fn rng_option<'a, T>(&mut self, options: &'a Vec<T>) -> Option<&'a T> {
```

Comment:
I've renamed this to `rng_choice`, since "option" is rather loaded in Rust. Also, I changed a bit the implementation to make it more ergonomic.